modules/ROOT/pages/mule-high-availability-ha-clusters.adoc

```diff
@@ -55,20 +61,20 @@ All Mule instances access the same Mule file folders concurrently, which can lea
 All Mule instances get the same TCP requests and then process duplicate messages.
 
 * JMS topics. +
-All Mule instances connect to the same JMS topic, which might lead to the repeated processing of messages when scaling the non-clustered Mule instance horizontally.
+All Mule instances connect to the same JMS topic, which might lead to the repeated processing of messages when scaling the non-clustered Mule instance horizontally. To the JMS broker the instances appear as separate subscribers and will all get a copy of each message, which is rarely the required behavior. Although there is a "shared subscriber" configuration available which might also address this requirement to instruct the JMS broker to treat all instances as a combined subscriber and to give them each separate messages.
```

Comment:
```suggestion
All Mule instances connect to the same JMS topic, which might lead to the repeated processing of messages when scaling the non-clustered Mule instance horizontally. To the JMS broker, the instances appear as separate subscribers, all of which get a copy of each message. This behavior is rarely required, but a "shared subscriber" configuration is available to instruct the JMS broker to treat all instances as a combined subscriber and to give them each separate messages.
```

Reply:
The proposed split of the sentences with "but" has changed the meaning: "This behaviour is rarely required" bit now seems to refer to the wrong part.. The rarely wanted behaviour is the "all nodes get the same messages", NOT the shared subscriber functionality. 
As in it would be pretty weird to want all 4 nodes to each get the same messages to process - you'd almost always want to use the shared subscriber functionality to make sure each of the nodes gets different messages.

Reply:
@eze210 
All Mule instances connect to the same JMS topic, which might lead to the repeated processing of messages when scaling the non-clustered Mule instance horizontally. To the JMS broker, the instances appear as separate subscribers, all of which get a copy of each message. This behavior is rarely required. To avoid this scenario,  a "shared subscriber" configuration is available to instruct the JMS broker to treat all instances as a combined subscriber and to give them each separate messages.